.circleci/config.yml

```diff
@@ -47,6 +47,9 @@ jobs:
           region: us-east-1
       - run: >-
           eval "$(aws configure export-credentials --profile default --format env)" 2> /dev/null
+      - run:
+          name: Build binary to scan
+          command: go install
```

Comment:
I don't believe this will work for what we need.  We need to have all the dependencies available in the source code.  Could you please try using `go mod vendor` instead